src/System.Xml.XmlSerializer/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -637,6 +637,16 @@ public static XmlSerializer[] FromTypes(Type[] types)
         {
             if (types == null)
                 return Array.Empty<XmlSerializer>();
+
+#if NET_NATIVE
+            var serializers = new List<XmlSerializer>();
```

Comment:
cc @shmao 

If we end up taking a change here lets file it as a separate issue and not fix it as part of the mirror.